src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Please add a private String property ```color```, and a constructor which will accept a single parameter as a value of this property. 
Also please rename enum values using constant case, and use a lowercase color names as value of ```color``` property.